src/h5web/visualizations/containers/NxImageContainer.tsx

```diff
@@ -10,17 +9,14 @@ function NxImageContainer(props: VisContainerProps): ReactElement {
   const { entity } = props;
   assertGroup(entity);
 
-  const nxDataGroup = findNxDataGroup(entity);
-  assertDefined(nxDataGroup, 'Expected to find NXdata group');
```

Comment:
Here, the `entity` should already be an `NXdata` group! 🎉 We make sure of it with an assertion in `useNxData`.

Reply:
I wonder if it might be clearer to move the assertion out of `useNxData`

Reply:
We can't type check it, so it's more of a fail safe, and I thought it safer to have this fail safe inside `useNxData` so we don't forget the next time we add a NeXus visualization.